app.js

```diff
@@ -103,6 +103,10 @@ function openSecondaryWindow(windowName) {
       width: 1366,
       height: 768,
       show: false,
+      webPreferences: {
+        nodeIntegration: true,
```

Comment:
This is technical bad security wise. Need to revisit how the 2 processes communicate.